### PR TITLE
[TASK] Remove templating tutorial from Settings.cfg

### PR DESCRIPTION
**Pull request by linawolf**
As information got moved to the sitepackage tutorial

Even though it was not commented out it was not used anymore in the code

### CODE REVIEW

File: Documentation/NextSteps/Index.rst [hunk 1]
```diff
@@ -8,6 +8,7 @@ Next steps
 ==========
 
 If your activities are not limited to editorial work, you may want to further
-learn about TYPO3 CMS by following the :doc:`TypoScript in 45 minutes Tutorial
```
Comment (alexander-nitsche) [inline Documentation/NextSteps/Index.rst:11]: Do you remove the reference to the "TypoScript in 45 minutes" tutorial by intention?

File: Documentation/Settings.cfg [hunk 2]
```diff
@@ -37,7 +37,7 @@ t3coreapi      = https://docs.typo3.org/m/typo3/reference-coreapi/main/en-us/
 # t3home         = https://docs.typo3.org/
 # t3install      = https://docs.typo3.org/m/typo3/guide-installation/main/en-us/
 t3l10n         = https://docs.typo3.org/m/typo3/guide-frontendlocalization/main/en-us/
-# t3sitepackage  = https://docs.typo3.org/m/typo3/tutorial-sitepackage/main/en-us/
+t3sitepackage  = https://docs.typo3.org/m/typo3/tutorial-sitepackage/main/en-us/
 t3start        = https://docs.typo3.org/m/typo3/tutorial-getting-started/main/en-us/
 # t3tca          = https://docs.typo3.org/m/typo3/reference-tca/main/en-us/
 t3templating   = https://docs.typo3.org/m/typo3/tutorial-templating/main/en-us/
```
Comment (alexander-nitsche) [inline Documentation/Settings.cfg:43]: `t3templating` should be removed, not?

**Review by alexander-nitsche — CHANGES_REQUESTED** [review]

**Comment by alexander-nitsche** [comment]
Relates: https://github.com/TYPO3-Documentation/T3DocTeam/issues/190

File: Documentation/NextSteps/Index.rst (hunk 1 above)
Comment (linawolf) [reply Documentation/NextSteps/Index.rst:11]: Yes, the sitepackage contains the complete Information on how to start a site package. You wouldn't get ppl started by teaching them TypoScript form the scratch today...

**Review by alexander-nitsche — APPROVED** [review]

**Pull request merged**